pkg/operator/ceph/object/user.go

```diff
@@ -185,6 +198,11 @@ func DeleteUser(c *Context, id string, opts ...string) (string, error) {
 		if code, ok := exec.ExitStatus(err); ok && code == int(syscall.ENOENT) {
 			return result, nil
 		}
+
+		res, er := ListUserBucket(c, id)
+		if er == nil && res != "" && res != "[]" {
+			return result, errors.Wrap(err, fmt.Sprintf("failed to delete s3 user with buckets %q", res))
```

Comment:
What does the output look like when this error happens? Is it just the bucket names? Or is it a lot more information about the buckets? Seems like we just need the bucket name(s), right?

Reply:
This will list only the bucket names.